### PR TITLE
Worfklow implemented

### PR DESCRIPTION
**Pull request by dawidkeshop**

### CODE REVIEW

File: src/CaptainHook.Storage.Cosmos/QueryBuilders/SubscriberQueryBuilder.cs [hunk 1]
```diff
@@ -29,7 +29,7 @@ public CosmosQuery BuildSelectAllSubscribers()
         public CosmosQuery BuildSelectSubscriber(SubscriberId subscriberId)
         {
             var query = new QueryDefinition(@"select * from c where c.id = @id")
-                .WithParameter("@id", subscriberId);
+                .WithParameter("@id", subscriberId.ToString());
```
Comment (mszymeshop) [inline src/CaptainHook.Storage.Cosmos/QueryBuilders/SubscriberQueryBuilder.cs:32]: nit: Is subscriber guaranteed not to be null?

File: src/CaptainHook.Api/Controllers/EventsController.cs [hunk 2]
```diff
@@ -27,16 +29,29 @@ public EventsController(IMediator mediator, IBigBrother bigBrother)
...
-        [ProducesResponseType(typeof(ErrorBase), StatusCodes.Status409Conflict)]
+        [ProducesResponseType(typeof(DirectorServiceIsBusyError), StatusCodes.Status409Conflict)]
+        [ProducesResponseType(typeof(ErrorBase), StatusCodes.Status500InternalServerError)]
         public async Task<IActionResult> PutWebhook([FromRoute] string eventName, [FromRoute] string subscriberName, [FromBody] EndpointDto dto)
         {
             var request = new UpsertWebhookRequest(eventName, subscriberName, dto);
             var result = await _mediator.Send(request);
-            return Ok(result);
+
+            return result.Match<IActionResult>(
```
Comment (mszymeshop) [inline src/CaptainHook.Api/Controllers/EventsController.cs:44]: In the future, when the record is updated, what status code is the operation going to have?

File: src/CaptainHook.Storage.Cosmos/QueryBuilders/SubscriberQueryBuilder.cs (hunk 1 above)
Comment (tabman83) [reply src/CaptainHook.Storage.Cosmos/QueryBuilders/SubscriberQueryBuilder.cs:32]: Validation is performed at repository level in GetSubscriberAsync

File: src/CaptainHook.Api/Controllers/EventsController.cs (hunk 2 above)
Comment (dawidkeshop) [reply src/CaptainHook.Api/Controllers/EventsController.cs:44]: 204

**Pull request merged**